### ISSUE TITLE
Implement tests

### ISSUE THREAD

**Issue by sarahmonster** [a8cteam51/strikestart#18]
Tests are good, so we should probably write some at some point;

**Comment by danielwrobert** [comment]
Good call. I had dug into testing React components not so long ago while planning to implement them in Strikebase as well. I'd love to take this one!
